framework/include/userobject/RadialAverage.h

```diff
@@ -0,0 +1,153 @@
+/**********************************************************************/
+/*                     DO NOT MODIFY THIS HEADER                      */
+/* MAGPIE - Mesoscale Atomistic Glue Program for Integrated Execution */
```

Comment:
Not the right header, but it would be nice if you could preserve the authorship of the underlying code. This could have been done (and could still be done) by creating a patch set from within magpie to export the files you based this code on, applying that patch set to MOOSE, and then performing git renames and copying your modified versions over the code.

Reply:
Will do. Is there an example of something that did that so I can recreate it?

Reply:
Here are the instructions from the Marmot wiki:

In the marmot directory:

```
rm -rf /tmp/mergepatches
mkdir /tmp/mergepatches
export reposrc='FILE1 FILE2 [...] DIR1 DIR2 [...]'
```
If you are using the zsh shell (default for Mac OS Catalina or later):
```
git format-patch -o /tmp/mergepatches $(git log ${=reposrc}|grep ^commit|tail -1|awk '{print $2}')^..HEAD ${=reposrc}
```
If you are using the bash shell (default on most other systems):
```
git format-patch -o /tmp/mergepatches $(git log $reposrc|grep ^commit|tail -1|awk '{print $2}')^..HEAD $reposrc
```

In the moose directory:

```
git am --ignore-whitespace --whitespace=nowarn --directory='modules/phase_field/' /tmp/mergepatches/00*.patch
```